### PR TITLE
added sh_history

### PR DESCRIPTION
**Pull request by PickmanSec**
2nd time is the charm. I added the sh_histroy file to shell history.

### CODE REVIEW

**Comment by osqueryer** [comment]
Can one of the osquery admins reply with "ok to test" to kick off a build...

File: osquery/tables/system/shell_history.cpp [hunk 1]
```diff
@@ -23,7 +23,7 @@ namespace osquery {
 namespace tables {
 
 const std::vector<std::string> kShellHistoryFiles = {
-    ".bash_history", ".zsh_history", ".zhistory", ".history",
+    ".bash_history", ".zsh_history", ".zhistory", ".history",".sh_history",
```
Comment (theopolis) [inline osquery/tables/system/shell_history.cpp]: Just a small nit, no change needed, but there should be a space between the comma and start of the string literal.

If you run `make format` after `git add .` clang-format will apply the recommended styling. :)

**Comment by theopolis** [comment]
ok to test

**Comment by osqueryer** [comment]
Build (87d0216296d356d30dbf135ffb0be068069a841a) is good! :+1:

**Pull request merged**